lib/camunda-cloud/Modeler.js

```diff
@@ -48,6 +52,8 @@ inherits(Modeler, BaseModeler);
 Modeler.prototype._camundaCloudModules = [
   modelingModule,
   contextPadModule,
+  disableCollapsedSubprocessModule,
+  drilldownModule,
```

Comment:
Why do we include the drilldown module in the first place if we disable it? Is it necessary?

Reply:
The drilldown module is shipped with the base modeler. The one we include as `_camundaCloudModules` overrides the default one with `null`

Reply:
Is there another way to disable modules? I could not find anything about it

Reply:
Makes sense. Thanks for clarifying. Disabling modules using empty modules with the same key is fine. The only alternative would be monkey patching 🐒 .